erpnext/hr/doctype/employee/employee.json

```diff
@@ -2418,6 +2418,36 @@
    "search_index": 0, 
    "set_only_once": 0, 
    "unique": 0
+  }, 
+  {
+   "allow_bulk_edit": 0, 
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "columns": 0, 
+   "fieldname": "has_subordinates", 
```

Comment:
no need for has subordinates. You should be able to add Employees against any other employee